app/download/download.js

```diff
@@ -54,45 +54,16 @@ app.controller('DownloadCtrl', function($scope, $http, $uibModalInstance, $locat
 
 
   function downloadFile(fileName, data, strMimeType) {
```

Comment:
It doesn't look like `strMimeType` is ever passed to the function so it always defaults to `application/octet-stream;charset=utf-8`

It might be a good idea to define the mime type on a format basis in hardcoded-data.js